myapp/app/controllers/tasks_controller.rb

```diff
@@ -1,6 +1,11 @@
 class TasksController < ApplicationController
   def index
     @tasks = Task.all
```

Comment:
maybe not relevant to the task this time, but does the training ask you to do pagination in upcoming tasks? Usually we don't pull all records.

Reply:
Based on the training specifications, none of the task steps require pagination.

Reply:
just double checked again, the pagination is at [step 14](https://github.com/Fablic/training/blob/develop/steps_en.md#step-14-lets-add-pagination). Let's add it later~